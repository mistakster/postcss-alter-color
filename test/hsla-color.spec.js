const process = require('./utils/process');

it('should alter hsla colors', () => {
  const source = `
div {
  color: hsla(0,0%,0%,1);
}
`;

  return process(source, {from: 'black', to: 'red'})
    .andMatchSnapshot();
});

it('should alter hsla color in a complex value', () => {
  const source = `
div {
  color: hsla(0,0%,0%,1);
  border: 1px solid hsla(0,0%,100%,1);
  outline: 1px solid hsla(0,0%,0%,1);
}
`;

  return process(source, {from: 'black', to: 'red'})
    .andMatchSnapshot();
});

it('should alter hsla color and opacity in a complex value', () => {
  const source = `
div {
  color: hsla(0,0%,0%,0.5);
  border: 1px solid hsla(0,0%,100%,1);
  outline: 1px solid hsla(0,0%,0%,1);
}
`;

  return process(source, {from: 'black', to: 'red'})
    .andMatchSnapshot();
});
