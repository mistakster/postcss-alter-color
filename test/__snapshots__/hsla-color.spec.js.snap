// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should alter hsla color and opacity in a complex value 1`] = `
"
div {
  color: hsla(0,0%,0%,0.5);
  border: 1px solid hsla(0,0%,100%,1);
  outline: 1px solid hsla(0,100%,50%,1);
}
"
`;

exports[`should alter hsla color in a complex value 1`] = `
"
div {
  color: hsla(0,100%,50%,1);
  border: 1px solid hsla(0,0%,100%,1);
  outline: 1px solid hsla(0,100%,50%,1);
}
"
`;

exports[`should alter hsla colors 1`] = `
"
div {
  color: hsla(0,100%,50%,1);
}
"
`;
